test/test_coverage.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+# Create a link for the code to current dir
+ln -s src/main/scala/org ./
```

Comment:
Although I think this is a bad idea, I change it in later commit. Think if you write a program and when you finish it you will collect the data. But if you it move here, it looks like we don't collect data. People can test coverage on his own machine, and they need covaregeAggregate.